### ISSUE TITLE
PR review request in url breaks the extension

### ISSUE THREAD

**Issue by TimvdLippe** [dominique-mueller/web-extension-github-travis-status#17]
If a link to a PR review request is included in the url, the extension does not load. For example https://github.com/Polymer/tools/pull/281#pullrequestreview-116679767 does not work, while https://github.com/Polymer/tools/pull/281 works. Note that https://github.com/Polymer/tools/pull/281#issuecomment-385770571 also does not work, so I think only a direct link works.

I think https://github.com/dominique-mueller/web-extension-github-travis-status/blob/a5b462ebf7d8f139dd7cd7800792cc360b57e507/src/background/background.ts#L65 should be updated to
```ts
/^https:\/\/github\.com\/.+\/.+\/pull\/\d+.+$/.test( url );
```
or
```ts
/^https:\/\/github\.com\/.+\/.+\/pull\/\d+/.test( url );
```

**Comment by itsdevdom** [comment]
Thanks for opening the issue, seems you're right. Feel free to open a PR, otherwhise I'll work on it on some of the next few days.

**Comment by itsdevdom** [comment]
Just published version 1.1.1, should be available in the Chrome Web Store within the next few minutes.

**Comment by TimvdLippe** [comment]
I can confirm it has been fixed 🎉
